### PR TITLE
🔀🌐 [main, emv] update en_GB translations

### PR DESCRIPTION
**Pull request by Gooochy**

### CODE REVIEW

File: src/modules/extendedCallWindow/i18n/en_GB.js [hunk 1]
```diff
@@ -79,6 +79,10 @@ module.exports = {
                 0, 1, 2, 3, 4, 6, 7, 14, 15, 16, 17, 18, 23, 26, 35, 36, 37, 38,
                 39, 40,
             ],
+            [/^more firefighter$/u]: [
+                0, 1, 2, 3, 4, 6, 7, 14, 15, 16, 17, 18, 23, 26, 35, 36, 37, 38,
+                39, 40,
+            ],
```
Comment (jxn-30) [inline src/modules/extendedCallWindow/i18n/en_GB.js:85]: use `[/^more firefighters?$/u]: [` in line 78 instead of adding these 4 lines.

**Comment by Gooochy** [comment]
Not sure what happened with name so its now called patch 59 sorry 😞

File: src/modules/extendedCallWindow/i18n/en_GB.js (hunk 1 above)
Comment (Gooochy) [reply src/modules/extendedCallWindow/i18n/en_GB.js:85]: Sorry I dont understand what your asking, you want me to condense it to one line?

Comment (jxn-30) [reply src/modules/extendedCallWindow/i18n/en_GB.js:85]: There already is a list for plural. You can add singluar by changing the regex (key). That is cleaner and easier to maintain than adding singular as an extra list.

Comment (Cr4zyc4k3) [reply src/modules/extendedCallWindow/i18n/en_GB.js:85]: `^more firefighter(s?)$/u` should do the job

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**